src/lib/holocene/api-pagination.svelte

```diff
@@ -78,100 +132,151 @@
-          </button>
+      {/if}
+      <div class="flex items-center justify-center gap-3">
+        <button
+          class="caret"
+          disabled={!$store.hasPrevious}
+          on:click={store.previousPage}
+        >
+          <span
+            class="arrow arrow-left"
+            class:arrow-left-disabled={!$store.hasPrevious}
+          />
+        </button>
+        <div class="flex gap-1">
           <p>
-            {$store.currentPageNumber}–{$store.endingPageNumber}
+            {$store.indexStart}–{$store.indexEnd}
           </p>
-          <button
-            class="caret"
-            disabled={!$store.hasNext}
-            on:click={store.nextPage}
-          >
-            <Icon name="chevron-right" />
-          </button>
+          {#if total}
+            <p>
+              of {total}
+            </p>
+          {/if}
         </div>
-        <slot name="action-top-right" />
-      </nav>
-    </div>
-    {#if $store.loading}
-      <SkeletonTable rows={15} />
-    {:else if !isEmpty}
-      <slot visibleItems={$store.items} initialItem={[]} />
-    {/if}
-
-    <nav
-      class={`flex ${
-        $$slots['action-bottom-left'] ? 'justify-between' : 'justify-end'
-      }`}
-    >
-      <slot name="action-bottom-left" />
-      <div class="flex gap-4">
-        <FilterSelect
-          label="Per Page"
-          parameter={$store.key}
-          value={String($store.pageSize)}
-          {options}
-        />
-        <div class="flex items-center justify-center gap-1">
+        <button
+          class="caret"
+          disabled={!$store.hasNext}
+          on:click={fetchIndexData}
+        >
+          <span
+            class="arrow arrow-right"
+            class:arrow-right-disabled={!$store.hasNext}
+          />
+        </button>
+      </div>
+      <slot name="action-top-right" />
+    </nav>
+  </div>
+  {#if $store.loading}
+    <SkeletonTable rows={15} />
+  {:else}
+    <slot
+      updating={$store.updating}
+      visibleItems={$store.visibleItems}
+      activeIndex={$store.activeIndex}
+      setActiveIndex={store.setActiveIndex}
+    />
+  {/if}
+  <nav
+    class={`flex ${
+      $$slots['action-bottom-left'] ? 'justify-between' : 'justify-end'
+    }`}
+  >
+    <slot name="action-bottom-left" />
+    <div class="flex gap-4">
+      {#if $store.visibleItems.length}
+        {#if pageSizeOptions.length}
+          <FilterSelect
+            label="Per Page"
+            parameter={$store.key}
+            value={String($store.pageSize)}
+            options={pageSizeOptions}
+          />
+        {/if}
+        <div class="flex items-center justify-center gap-3">
           <button
             class="caret"
             disabled={!$store.hasPrevious}
             on:click={store.previousPage}
           >
-            <Icon name="chevron-left" />
+            <span
+              class="arrow arrow-left"
+              class:arrow-left-disabled={!$store.hasPrevious}
+            />
```

Comment:
Why move away from the chevrons? Alternatively, could we use the `<Shortcut />` component here?

Reply:
I wanted something that looked like the arrow keys. Let me look into using the Shortcut